apps/files/src/components/Collaborators/EditCollaborator.vue

```diff
@@ -1,6 +1,15 @@
 <template>
   <div class="files-collaborators-collaborator-edit-dialog">
-    <div v-if="user.id !== collaborator.owner.name" class="uk-text-meta uk-flex uk-flex-middle uk-margin-small-bottom"><oc-icon name="repeat" class="uk-margin-small-right" /> {{ collaborator.owner.displayName }}</div>
+    <transition enter-active-class="uk-animation-slide-top-small" leave-active-class="uk-animation-slide-top-small uk-animation-reverse"
+                name="custom-classes-transition">
+      <div class="uk-alert-danger" uk-alert v-if="errors">
```

Comment:
There's an ODS alert component - https://owncloud.github.io/owncloud-design-system/#/Elements/oc-alert

Reply:
🙈 thanks! edited the commit and also included the alert component in the public link editing component.